src/component/fetchdata.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
```

Comment:
As much as possible, kindly avoid disabling linter checks inline. They represent coding standards you share with your team.